tests/defer/src/commonTest/kotlin/test/DeferNormalizedCacheTest.kt

```diff
@@ -75,7 +76,7 @@ class DeferNormalizedCacheTest {
         """{"incremental": [{"data":{"cpu":"386","year":1993,"screen":{"__typename":"Screen","resolution":"640x480"}},"path":["computers",0]}],"hasNext":true}""",
         """{"incremental": [{"data":{"isColor":false},"path":["computers",0,"screen"],"label":"a"}],"hasNext":false}""",
     )
-    mockServer.enqueueMultipart(jsonList)
+    mockServer.enqueueMultipart("application/json").enqueueStrings(jsonList)
```

Comment:
The parts content-type is now required because it felt like it belonged next to the data that is sent. If we wanted to make things even easier we could add a `enqueueJsonMultipart` or so but I don't think it's worth it?